libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFAuthenticationManager.m

```diff
@@ -1164,23 +1182,4 @@ - (void)identityCoordinator:(SFIdentityCoordinator *)coordinator didFailWithErro
     }
 }
 
-#pragma mark - UIAlertViewDelegate
-
-- (void)alertView:(UIAlertView *)alertView didDismissWithButtonIndex:(NSInteger)buttonIndex
-{
-    if (alertView == _statusAlert) {
-        _statusAlert = nil;
```

Comment:
This got lost in the new functionality.  Which means that the Retry functionality will only ever work once, because of the not-`nil` check in `showAlertWithTitle:message:buttonTitle:tag:`